ArcGISRuntimeSDKQt_QMLSamples/CloudAndPortal/ShowOrgBasemaps/ShowOrgBasemaps.qml

```diff
@@ -267,7 +267,7 @@ Rectangle {
     /*
     // Uncomment this section when running as standalone application
     AuthenticationView {
-        authenticationManager: AuthenticationManager
+        anchors.fill: parent
     }
     */
```

Comment:
@jared-2016 can you confirm if this is still needed or not when running samples individually? This is currently commented out. There was a discussion at one point that these wouldn't be needed anymore, but I don't recall the specifics so I could be wrong on that.

Reply:
@JamesMBallard Yes the `AuthenticationView` is still needed for samples to be run individually.

There is another discussion about if the sample viewer apps would need one as well if they were defined in the individual samples but that is less clear as some samples which don't normally require one might need one in some rare scenarios. Lucas touched on this briefly in the sample viewer PR if you want to take a look there.